.github/workflows/python-app.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ monaparty-develop ]
+  pull_request:
+    branches: [ monaparty-develop ]
```

Comment:
only monaparty-develop branch?

Reply:
Yes only on. Because we shouldn't apply any patches on `monaparty` branch excludes critical hotfixes.
Basically issues must be fixed in the develop branches before merging to the master branch.

Reply:
Personally, I think it's better to include the `monaparty` branch, but I'll follow the policy.